src/sota_tools/accumulator.h

```diff
@@ -40,42 +40,42 @@ class accumulator_type : public boost::program_options::value_semantic {
     return this;
   }
 
-  virtual std::string name() const { return std::string(); }
+  virtual std::string name() const { return std::string(); } // NOLINT
```

Comment:
What's the complaint here that's being bypassed?

Reply:
clang-tidy complains about virtual and recommends virtual. But some of these methods seem to only exist in the parent class in certain versions of boost (I had different warnings on my machine and CI) and marking a method as overriding while it's not is an error.

Would probably be better to ignore this file altogether since it's not our code but I took the easy route this time.